src/ontology/config/property-map-1.sssom.tsv

```diff
@@ -1,2 +1,4 @@
 subject_id	object_id
 http://purl.org/dc/elements/1.1/source	oboInOwl:source
+http://www.w3.org/2004/02/skos/core#prefLabel	http://www.w3.org/2000/01/rdf-schema#label
```

Comment:
Used to update some properties of same/similar type that we don't want to use, with properties that we do want to use.